examples/ethereum/list-rewards/main.go

```diff
@@ -12,13 +12,23 @@ import (
 	"log"
 	"time"
 
-	"google.golang.org/api/iterator"
-
 	"github.com/coinbase/staking-client-library-go/auth"
 	"github.com/coinbase/staking-client-library-go/client"
 	"github.com/coinbase/staking-client-library-go/client/options"
-	rewardsV1 "github.com/coinbase/staking-client-library-go/client/rewards/v1"
-	rewardspb "github.com/coinbase/staking-client-library-go/gen/go/coinbase/staking/rewards/v1"
+	"github.com/coinbase/staking-client-library-go/client/protocols"
+	"github.com/coinbase/staking-client-library-go/client/rewards/reward"
+	api "github.com/coinbase/staking-client-library-go/gen/go/coinbase/staking/rewards/v1"
```

Comment:
nit: we inconsistently import "github.com/coinbase/staking-client-library-go/gen/go/coinbase/staking/rewards/v1" with api, rewardspb or stakingpb. Would be nice to stick to one.

Reply:
Yeh definitely. I'll standardize on `api` for now.